scripts/fdbshow

```diff
@@ -139,10 +175,11 @@ def main():
                                      formatter_class=argparse.RawTextHelpFormatter)
     parser.add_argument('-p', '--port', type=str, help='FDB learned on specific port: Ethernet0', default=None)
     parser.add_argument('-v', '--vlan', type=str, help='FDB learned on specific Vlan: 1001', default=None)
+    parser.add_argument('-f', '--fast', action='store_true', help='Display large number of FDB entries fastly')
```

Comment:
>fastly [](start = 103, length = 6)

fast

Reply:
Thank you. I have changed codes as reviewed.